### DISCUSSION TITLE
Import Issues

### DISCUSSION THREAD

**Discussion by twinsmom71** [snipe/snipe-it#9235] [Q&A] [answered]
I have tried multiple times to import a .csv file that contains 100 assets.  The file imports with no errors and tells me success every time, yet only 69 assets get loaded, every time it's 69.  I used the sample import .csv file as a starter.  I have double and triple checked field formats.  All import fields match perfectly prior to import.  Any ideas what the issue could be?

**Comment by Canpy** [comment, marked as answer]
i have the same problem i uploaded a file and only 1 of 24 records gets imported

**Reply by twinsmom71** [reply]
I figured out my issue, I knew Asset Tag field was required but did not realize you cannot have any duplicates.  It was dropping any duplicates.

**Reply by Canpy** [reply]
Yes, I solved my problem too, I didn't even know that asset tags were needed.
